spec/lib/post_handler_spec.rb

```diff
@@ -92,6 +93,60 @@
         }
       end
     end
+
+    describe "when changed ownership of the post" do
+      describe "from category expert to another expert" do
```

Comment:
For this, if you're transfering from one expert to another, wouldn't you want the custom field to still be present?